Gopkg.toml

```diff
@@ -15,3 +15,7 @@ required = [
 [[prune.project]]
   name = "github.com/google/licenseclassifier"
   non-go = false
+
+[[constraint]]
+  name = "cloud.google.com/go"
+  version = "0.39.0"
```

Comment:
I'm not seeing this version in eventing https://github.com/knative/eventing/blob/master/Gopkg.toml
Is it the latest release version?

Reply:
The latest version was `v0.39.0` when eventing updated `cloud.google.com/go` a couple of days ago, but it's now `v0.40.0`. I added this constraint to ensure the consistency. But again, as I mentioned in https://github.com/knative/test-infra/issues/957, I'm not sure if it's the best practice, and I'm even not sure if we should update dependencies here in this repo. 
Any thoughts or better ideas for this?

Reply:
You shouldn't need to update it here, unless you're using some new feature in eventing that's conflicting with code here (or avoiding a particular version). Is it the only solution?

Reply:
I also tried to directly update dependencies in `eventing` and it looked fine, see https://github.com/knative/eventing/pull/1411. 
So maybe this PR is not needed?
Such issues are unavoidable if we have different version of dependencies in each project, unless there is one single place managing the dependencies for all repos under Knative projects, but it seems impossible under the current dependency model.

Reply:
If you solved the issue in eventing, so I don't see why unnecessarily restricting a version here.

Reply:
Yeah that's true, I'll close this PR.